tests/test/slgext.rs

```diff
@@ -103,13 +103,49 @@ fn flounder() {
 
         goal {
             exists<T> { not { T: A } }
-        } yields_first[SolverChoice::SLG { max_size: 10 }] {
+        } yields_first[SolverChoice::slg(10, None)] {
             // FiXME(jackh726): need to refactor framework to detect flouder/ambiguous
             "for<?U0> { substitution [?0 := ^0], lifetime constraints [] }"
         }
     }
 }
 
+// Test that, when solving `?T: Sized`, we only wind up pulling a few
+// answers before we stop.
+// This is similar to the `breadth_first` test, except the order of the 
+// FIXME: This is basically the same as `breadth_first`. Is it testing something different?
```

Comment:
I think the only difference is the order of the impls, which can however be significant (e.g., which program clause do we try first?)

Reply:
Right. I guess I just don't know if this is trying to test something different or not.